go/obscuronode/enclave/enclave.go

```diff
@@ -248,23 +251,24 @@ func (e *enclaveImpl) RoundWinner(parent obscurocommon.L2RootHash) (nodecommon.E
 
 	parentState := e.storage.FetchRollupState(head.Hash())
 	// determine the winner of the round
-	winnerRollup, s := e.findRoundWinner(usefulRollups, head, parentState, e.storage, e.blockResolver)
+	winnerRollup, s, hasTxs := e.findRoundWinner(usefulRollups, head, parentState, e.storage, e.blockResolver)
```

Comment:
this hasTxs logic has to go because of that bug

Reply:
Done 👍